### PR TITLE
F/toast link

### PR DESCRIPTION
**Pull request by tgadre**
screen shot 2017-03-21 at 5 20 45 pm

##### **Description**
A new toast text input field to display links

##### **What did you change?**
1. Added a new text field, value is tield to new link property
2. New close icon which is tied to a new isCloseable property
3. Styling

##### **Reviewers**
* @jgodi
* @more

### CODE REVIEW

File: src/elements/toast/Toast.ts [hunk 1]
```diff
@@ -17,10 +17,16 @@ import { Component, Input, OnInit, OnChanges, SimpleChanges } from '@angular/cor
         <div class="toast-content">
             <h5 *ngIf="title">{{title}}</h5>
             <p *ngIf="message" [class.message-only]="!title">{{message}}</p>
+            <div class="link-generated">
```
Comment (jgodi) [inline src/elements/toast/Toast.ts]: ngIf on the div

File: src/elements/toast/Toast.ts [hunk 2]
```diff
@@ -17,10 +17,16 @@ import { Component, Input, OnInit, OnChanges, SimpleChanges } from '@angular/cor
...
+                <input *ngIf="link" type="text" [value]="link" onfocus="this.select();"/>
+            </div>
             <div class="dialogue">
                 <ng-content></ng-content>
             </div>
         </div>
+        <div class="close-icon" onclick="close()">
```
Comment (jgodi) [inline src/elements/toast/Toast.ts]: ngIf iscloseable

**Comment by jgodi** [comment]
Needs to have a demo attached to it

File: src/elements/toast/Toast.ts (hunk 1 above)
Comment (tgadre) [inline src/elements/toast/Toast.ts]: fixed.

File: src/elements/toast/Toast.ts (hunk 2 above)
Comment (tgadre) [inline src/elements/toast/Toast.ts]: fixed.

**Pull request merged**